templates/pupil.jade

```diff
@@ -15,16 +15,25 @@ block page_scripts
       });
     })();
   script.
```

Comment:
@nathakits move this script to its own file and load it in this block for this page only.